packages/gasket-plugin-docs-graph/plugin.js

```diff
@@ -0,0 +1,41 @@
+/* eslint no-sync: 0 */
+const path = require('path');
+const fs = require('fs');
+const write = require('util').promisify(fs.writeFile);
+
+module.exports = {
+  name: 'docs-graph',
+  hooks: {
+    async docsGenerate(gasket, docsConfigSet) {
+      const { docsRoot } = docsConfigSet;
+      const targetRoot = path.join(docsRoot, 'generated-docs');
+      const target = path.join(targetRoot, 'mermaid.md');
```

Comment:
I think `lifecycles.md` or `lifecycle-graphs.md` be a more appropriate name for the content, rather than the format.